### PR TITLE
Add Talks page with the links to the videos

### PR DESCRIPTION
**Pull request by vrom911**
Adding it as a separate page, also highlighting in the separate section on the main index page.
I remove the Explanation section for now from the highlights as it is still not that loaded with info.
I will bring it back when it will look more full. Anyway, there is other ways to access the explanations 🙂

### CODE REVIEW

File: docs/about/talks.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+On this page you will find a collection of our talks and workshop videos showcasing Juvix.
+
+In these videos we delve into the fascinating world of this powerful language,
+uncovering its core principles, and demonstrating real-world examples of Juvix
+applications.
+
+Gain valuable insights and inspiration from our presentations at various conferences.
+
+- ["Juvix"](https://youtu.be/T8jkl7_BMAY) _@Privacy Evolution 22_ by **Paul Cadman**
+- ["Juvix: Towards a Functional Programming Language for Decentralized Applications and Beyond"](https://youtu.be/YAOFCQTAhUI) _@ETHPrague2023_ by **Veronika Romashkina**
```
Comment (jonaprieto) [inline docs/about/talks.md:12]: Put this list inside a section like `## 2023`.

Let's use a format like the one described here: https://www.scribbr.com/apa-examples/speech/ for talks.

File: docs/index.md [hunk 2]
```diff
@@ -39,13 +39,24 @@ section](./overview.md).
...
+    [:octicons-arrow-right-24: Read the book](./explanations/README.md) -->
+
+-  :fontawesome-solid-book-open:{ .lg .middle } __Talks and Workshops__
+
+    ---
+
+    A collection of talks and workshop videos showcasing Juvix. Gain valuable
+    insights and inspiration from our presentations at various conferences.
+
+    core principles, and demonstrating real-world examples of Juvix applications
```
Comment (jonaprieto) [inline docs/index.md:57]: This part is disconnected from the previous paragraph.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "juvix-docs",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```
Comment (jonaprieto) [inline package-lock.json:6]: Remove this file

**Review by jonaprieto — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (vrom911) [reply docs/index.md]: This is by mistake. Fixed!

File: docs/about/talks.md (hunk 1 above)
Comment (vrom911) [reply docs/about/talks.md]: I used that. But kept names in the NAME SURNAME order for being less formal in a way 🙂

**Review by jonaprieto — APPROVED** [review]
👍

**Pull request merged**